tests/auxtel/test_latiss_cwfs_align.py

```diff
@@ -0,0 +1,42 @@
+import logging
```

Comment:
I don't think this unit test is doing anything. Consider removing it.

Reply:
Doesn't this verify that the script is executable at a minimal level?
In the workflow procedure we wrote there is a minimum requirement of having a test that proves it compiles etc.

Reply:
The way it is right now it is not testing that there is an executable. Also, I don't think we have an executable yet and the run method on the class is still empty. So, even if we had and executable, it would not do much. To see how to test that the executable exists take a look at this one:

https://github.com/lsst-ts/ts_standardscripts/blob/cf710a07381fcf854e7aea7236ba558a79cf6f1a/tests/auxtel/test_startup.py#L50-L53

In any case, I think the best now would be to remove the test and let it the way it is right now. We should give it some thought and think about how to implement the run method and then proceed to create the executable and the test.

Reply:
Ok done, but lets try to get something in place this week as a standard. With the operations rehearsal coming up we'll want them following the proper protocol.